src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2818,10 +2818,10 @@ type QueryType {
         allowKeepingRentedBicycleAtDestination: Boolean
 
         """
-        The cost of arriving at the destination with the rented bicycle, to discourage doing so.
+        The cost of arriving at the destination with the rented vehicle, to discourage doing so.
         Default value: 0.
         """
-        keepingRentedBicycleAtDestinationCost: Int
```

Comment:
I think we should do this but in a backwards-compatible way. The dev meeting will be a good place to discuss this.

Reply:
Maybe the first step would be to correctly change the description without changing the name?

Reply:
Indeed